Dummy_Database.md

```diff
@@ -9,15 +9,29 @@ create table if not exists books(barcode varchar(100) primary key, name varchar(
 
 create table if not exists users(username varchar(100) primary key,password varchar(100), firstname varchar(100),lastname varchar(100),address text, phone varchar(100),mailid varchar(100),usertype int);
 
-insert into if not exists books values('10101','Programming in C','James k Wick',500,5);
+insert into if not exists books values('9780134190563','The Go Programming Language',Alan A. A. Donovan and Brian W. Kernighan,400,8);
```

Comment:
Great work @RagulTW , can you  remove "if not exists" from these insert commands.